<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>382</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>382</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="listTable">
        <property name="columnCount">
         <number>4</number>
        </property>
        <column>
         <property name="text">
          <string>编号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>书名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>作者</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>价格</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="searchWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>书名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>作者</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="authorEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>价格</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="priceEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delButton">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flashButton">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
